Gemfile

```diff
@@ -43,6 +43,8 @@ end
 group :development do
   gem 'better_errors'
   gem 'binding_of_caller'
+  gem 'wraith', '~> 3.0'
+  gem 'phantomjs', '~> 2.0'
```

Comment:
As discussed over Slack:
* We previously avoided phantomjs because of security concerns
* It  previously downloaded the binary over http and didn't do a checksum on the downloaded thing (now works over https)
* It uses a different version to our managed environment, it's hard to keep the two in sync